getting_started/setup_vm/roles/ccf_dependencies/tasks/install.yml

```diff
@@ -59,4 +59,47 @@
   command: make install
   args:
     chdir: "{{ workspace }}/mbedtls-{{ mbedtls_dir }}/build"
-  become: true
\ No newline at end of file
+  become: true
+
+- name: Download FlatBuffers source
```

Comment:
Would it be cleaner to have this in a different Ansible file?

Reply:
Only if we're going to use it somewhere else, I think we can burn that bridge when we come to it.